client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -59,6 +59,13 @@ const WRITE_INTENT_DEFAULT_THEME_STYLE_VARIATION = 'white';
 const SiteIntent = Onboard.SiteIntent;
 const SiteGoal = Onboard.SiteGoal;
 
+function isLaunchpadIntent( intent: SiteIntent ) {
+	if ( intent === SiteIntent.Write || intent === SiteIntent.Build ) {
```

Comment:
Could the content of this method could shortened? ie, drop the if/else and do something like this?

`return intent === SiteIntent.Write || intent === SiteIntent.Build;`

Reply:
Sure, that sounds fine. I just copy-pasta'd up here when the conditional was used inline below.